tools/perfscale-audit/metric-client/metric-client.go

```diff
@@ -50,15 +50,15 @@ const (
 //	at all times.
 const (
 	vmiPhaseCount                 = `sum by (phase) (kubevirt_vmi_phase_count{})`
-	avgVirtAPICPUUsage            = `avg(sum(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-api.*", container!="",container!="POD"}[%ds]))by (pod))`
+	avgVirtAPICPUUsage            = `avg(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-api.*", container!="",container!="POD"}[%ds]))`
 	avgVirtAPIMemUsageInMB        = `avg(avg_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds]))/1024/1024`
-	minVirtAPIMemUsageInMB        = `min_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds])/1024/1024`
-	maxVirtAPIMemUsageInMB        = `max_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds])/1024/1024`
-	avgVirtControllerCPUUsage     = `avg(sum(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-controller.*", container!="",container!="POD"}[%ds])) by (pod))`
+	minVirtAPIMemUsageInMB        = `avg(min_over_time(container_memory_rss{pod=~"virt-api.*", container!="POD", container!=""}[%ds]))/1024/1024`
```

Comment:
We probably want to tell people we are taking an average.  I think `minVirtAPIMemUsageInMB` should be `avgMinVirtAPIMemUsageInMB`?  Same for `maxVirtAPIMemUsageInMB`.